### PR TITLE
Fixing icons for new version of font awesome

### PR DESCRIPTION
**Pull request by caijim**
New version of font-awesome = new icon classes needed

### CODE REVIEW

File: lms/static/coffee/src/instructor_dashboard/queries.coffee [hunk 1]
```diff
@@ -470,11 +470,11 @@ class EmailWidget
     @get_students (error, students) =>
       if error
         $broken_icon = $ _.template('<div class="done">
-          <i class="icon-warning-sign"></i> <%= label %></div>',
+          <i class="icon fa fa-exclamation-triangle"></i> <%= label %></div>',
           {label: gettext("Sorry, we're having a problem with this query.
             Please delete this row and try again.")})
         tr.children()[4].innerHTML = $broken_icon[0].outerHTML
-        return @show_errors error
+        return @show_errors errorpro
```
Comment (caesar2164) [inline lms/static/coffee/src/instructor_dashboard/queries.coffee]: this is extraneous?

**Comment by stvstnfrd** [comment]
@caesar2164 I defer to your judgement on this one. I'll cherry-pick it when you give the :+1:

**Comment by caesar2164** [comment]
:+1:

**Comment by stvstnfrd** [comment]
Cherry-picked at fc9f6c961cf8045ff789eb309d6a011a1a3c3881

**Pull request closed without merging**